src/WpfMath.Tests/TestResults/ParserTests.newLineAfterMatrix.(matrix(line 1)line 2).approved.txt

```diff
@@ -0,0 +1,186 @@
+{
+  "TextStyle": null,
+  "RootAtom": {
+    "MatrixCells": [
+      [
+        {
+          "MatrixCells": [
```

Comment:
It is strange that there're two matrix structures for this test; need to investigate.

Reply:
This is okay: the `\matrix` differs in alignment with what we expect from the `\\` matrix structure.